roles/chrony/tasks/install.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Install Chorny - Debian based distributions
+  tags: [chrony, chrony-install]
```

Comment:
Don't add `chrony-install` tag. If you run the chrony tag and it's not installed, it will be installed. So `chrony-install` is confusing.